doc/devel/cmd_interface_devel.rst

```diff
@@ -39,7 +39,7 @@ above example we have used the ``desc`` metadata which holds human readable
 description of the input. The ``mandatory`` flag forces Nipype to throw an
 exception if the input was not set. ``exists`` is a special flag that works only
 for ``File traits`` and checks if the provided file exists. More details can be
-found at :doc:`interface_specs`.
+found at :ref:`interface_specs`.
```

Comment:
This change doesn't do anything. Might as well revert to keep the diff clean, unless there's a good reason to prefer `:ref:`.